### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by andrewsi10**
## Change description

See [CET-2313](https://cortex1.atlassian.net/browse/CET-2313) for details.

## Type of change

- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [ ] Improvement (improves codebase without changing functionality)

## Related issues

> Fix [#1]()

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

[CET-2313]: https://cortex1.atlassian.net/browse/CET-2313?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/EntityPage/EntityPage.test.tsx [hunk 1]
```diff
@@ -13,23 +13,114 @@
...
+        <EntityPage />
+      </EntityProvider>,
+      cortexApi,
+      {
+        '/': rootRouteRef,
+      },
+    );
+    expect(await findByText('Test Scorecard 1')).toBeVisible();
+    expect(await findByText('Test Scorecard 2')).toBeVisible();
+    expect(await findByText('oncall != null')).toBeVisible();
```
Comment (cristinabuenahora) [inline src/components/EntityPage/EntityPage.test.tsx:110]: Could you use checkForText here?

File: src/components/EntityPage/EntityScorecardOverview/EntityScorecardOverview.test.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+              score: 0,
+            },
+          ],
+        }),
+      ]),
+  };
+
+  const entity = Fixtures.entity();
+
+  it('Properly renders EntityScorecardOverview', async () => {
```
Comment (cristinabuenahora) [inline src/components/EntityPage/EntityScorecardOverview/EntityScorecardOverview.test.tsx:91]:
```suggestion
  it('renders EntityScorecardOverview', async () => {
```

File: src/components/EntityPage/EntityScorecardRules/EntityScorecardRules.test.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+          score: 1,
+        },
+      ],
+      ladderLevels: [],
+    },
+  });
+
+  const entity = Fixtures.entity();
+
+  it('Properly renders EntityScorecardRules', async () => {
```
Comment (cristinabuenahora) [inline src/components/EntityPage/EntityScorecardRules/EntityScorecardRules.test.tsx:67]:
```suggestion
  it('renders EntityScorecardRules', async () => {
```

File: src/components/EntityPage/EntityScorecardRules/EntityScorecardRules.test.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    const { checkNotText, clickButtonByText, findByText } = renderWrapped(
+      <EntityProvider entity={entity}>
+        <EntityScorecardRules score={serviceScorecardScore} />
+      </EntityProvider>,
+    );
+    expect(await findByText('All Rules')).toBeVisible();
```
Comment (cristinabuenahora) [inline src/components/EntityPage/EntityScorecardRules/EntityScorecardRules.test.tsx:73]: same checkForText question here

**Comment by cristinabuenahora** [comment]
Thank you for writing these tests!! 👏

**Review by cristinabuenahora — APPROVED** [review]

**Review by FoodProduct — COMMENTED** [review]
🙏 📈

**Pull request merged**